src/ngraph/node.hpp

```diff
@@ -139,6 +148,41 @@ namespace ngraph
     public:
         virtual ~Node();
 
+        /// Tests if a node is of op type T
+        template <typename NodeType>
+        bool has_type() const
```

Comment:
I don't understand what `has_type` is. I do understand what `is_type` is but can you add a comment as to what this is. 
stl uses is_* all over the place

Reply:
Change to `is_type`